src/components/UserAuth/Login.jsx

```diff
@@ -0,0 +1,150 @@
+import React, {useState} from "react";
+import "../../styles/root.css";
+import {steve} from "../../assets/images";
+import {
+  Link,
+  Navigate,
+  useLocation,
+  useNavigate,
+} from "react-router-dom";
+//import {UseAuth} from "../../context/AuthProvider";
+import {useDispatch, useSelector} from "react-redux";
+import {loginUser} from "../../Redux/authSlice";
+import { useEffect } from "react";
+
+export function Login() {
+  const {authStatus, authError, token} = useSelector(
+    (state) => state.auth
+  );
+  const dispatch = useDispatch();
+  const location = useLocation();
+  const navigate = useNavigate();
+
+  const initialState = {
+    username: "",
+    password: "",
+  };
+  const [userDetails, setUserDetails] =
+    useState(initialState);
+
+  const loginHandler = (e, userDetails) => {
+    e.preventDefault();
+    setUserDetails({...userDetails});
+    const res = dispatch(loginUser({userDetails}));
+
+    console.log('res following', res)
```

Comment:
Suggestion: you can remove this console log on the final branch before deployment.